JetNews/gradle/libs.versions.toml

```diff
@@ -9,10 +9,7 @@ androidx-activity-compose = "1.9.0"
 androidx-appcompat = "1.7.0"
 androidx-benchmark = "1.2.4"
 androidx-benchmark-junit4 = "1.2.4"
-androidx-compose-bom = "2024.06.00"
```

Comment:
I think this changes can be reverted right? You should also change all the time files at once and not individually if making a change to them.

Reply:
Yeah, I made the temporary change to make sure it could build with pre-stable and then updated the bom to the same as main to verify it still worked correctly with the released libraries. I just merged with main to make it more clear though :)